src/titiler/core/titiler/core/factory.py

```diff
@@ -216,6 +218,13 @@ def add_route_dependencies(
                 route.dependencies.extend(dependencies)  # type: ignore
 
 
+def get_feature(
```

Comment:
Just wondering, if we trim lat/lngs to 7 or 8 places, will this be enforced here or from the Field Agent app/repo?

Reply:
That's done in FieldAgent. Probably use [TurfJS](https://turfjs.org/docs/#simplify) for trimming.